platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/Core.swift

```diff
@@ -12,6 +12,21 @@ enum PlatformResponse {
     case Example(String)
 }
 
+extension PlatformResponse: Encodable {
+    
+    private enum CodingKeys: String, CodingKey {
+        case Example
```

Comment:
So this used just to specify that the key should be uppercase? If so, then maybe the PlatformResponse from core side should decode the values in camelcase, like [this](https://github.com/polypoly-eu/polyPod/blob/8b8ac467fef4404c8681db551dbc65ff13c206bd/platform/core/src/user_session/mod.rs#L8)

Reply:
The problem is not the key `Example`. The enum `PlatformResponse.Example("Text")` gets encoded to:

`.map([.string("Example") : .map([.string("_0"): .string("Text")])])`

Swift encodes the arguments of the enum case by making a map from index to value. And Rust deserialisation can't deal with that.

Reply:
Ok, so it is about how Rust and Swift do handle the associated values without labels. Just as an idea, maybe we should add the proper associated value labels for both Rust and Swift. Instead of `case Example(String)` have `case Example(name:String)`

Reply:
🤔 I need to try that. It would save a ton of work.

Reply:
In Rust it does not work 

<img width="465" alt="Screenshot 2022-08-17 at 13 12 50" src="https://user-images.githubusercontent.com/25011107/185094099-7f27ac02-5fab-4a12-ab00-2afb8e63f4c8.png">

Reply:
Actually, that was the wrong syntax
<img width="439" alt="Screenshot 2022-08-17 at 13 14 37" src="https://user-images.githubusercontent.com/25011107/185094415-4fbeda75-0acf-43dc-af95-222f458af9b3.png">